.github/workflows/build-cross-compile.yml

```diff
@@ -186,3 +197,10 @@ jobs:
           make-target: 'hotspot ${{ inputs.make-arguments }}'
           platform: linux-${{ matrix.target-cpu }}
         if: steps.create-sysroot.outcome == 'success' || steps.get-cached-sysroot.outputs.cache-hit == 'true'
+
+      - name: 'Upload bundles'
+        uses: ./.github/actions/upload-bundles
+        with:
+          platform: linux-${{ matrix.target-cpu }}
+          debug-suffix: '${{ matrix.suffix }}'
```

Comment:
This shouldn't be required as there is no `matrix.suffix` in this workflow. I see I have done the same mistake in android and ios builds both of which do not have a matrix.